zebra-state/src/service/finalized_state.rs

```diff
@@ -246,12 +274,25 @@ impl FinalizedState {
             batch.zs_insert(height_by_hash, hash, height);
             batch.zs_insert(block_by_height, height, &block);
 
-            // TODO: sprout and sapling anchors (per block)
-
             // "A transaction MUST NOT spend an output of the genesis block coinbase transaction.
             // (There is one such zero-valued output, on each of Testnet and Mainnet .)"
             // https://zips.z.cash/protocol/protocol.pdf#txnconsensus
             if block.header.previous_block_hash == GENESIS_PREVIOUS_BLOCK_HASH {
+                batch.zs_insert(
```

Comment:
We might want to explain that these are the empty note commitment trees, and how we make sure they're not accidentally used to validate transactions too early.

Reply:
Done in a4df422ed